### DISCUSSION TITLE
Option to specify number of waves in a raid

### DISCUSSION THREAD

**Discussion by ClinShaiju** [PurpurMC/Purpur#1565] [Feature Requests]
Is it possible to add an option to specify how many waves of a raid spawns, regardless of the difficulty?

I would like to keep the difficulty at Easy while also allowing evokers to spawn during raids for totems, though I couldn't find any existing way to do this.

The suggestion in #900 would also be able to achieve this, however I feel the number of waves is a much simpler solution.

**Comment by Krakenied** [comment]
not that easy to implement, actually #900 seems to be easier due to the fact how it works rn (its a really weird array)
```java
VINDICATOR(EntityType.VINDICATOR, new int[]{0, 0, 2, 0, 1, 4, 2, 5}), EVOKER(EntityType.EVOKER, new int[]{0, 0, 0, 0, 0, 1, 1, 2}), PILLAGER(EntityType.PILLAGER, new int[]{0, 4, 3, 3, 4, 4, 4, 2}), WITCH(EntityType.WITCH, new int[]{0, 0, 0, 0, 3, 0, 0, 1}), RAVAGER(EntityType.RAVAGER, new int[]{0, 0, 0, 1, 0, 1, 0, 2});
```
